lib/vtls/schannel.c

```diff
@@ -884,14 +884,13 @@ schannel_recv(struct connectdata *conn, int sockindex,
                            size, &nread);
     /* check for received data */
     if(*err != CURLE_OK)
-      ret = -1;
+      return -1;
     else {
```

Comment:
It's fair feedback. I'm following the pattern used in the surrounding code, but I'm happy either way:
https://github.com/bagder/curl/blob/master/lib/vtls/schannel.c#L868
https://github.com/bagder/curl/blob/master/lib/vtls/schannel.c#L952

Please let me know which way the team prefers, and I'll update the pull request.